pkg/controller/argocd/rolebinding.go

```diff
@@ -41,10 +41,9 @@ func newClusterRoleBindingWithname(name string, cr *argoprojv1a1.ArgoCD) *v1.Clu
 func newRoleBinding(cr *argoprojv1a1.ArgoCD) *v1.RoleBinding {
 	return &v1.RoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        cr.Name,
-			Labels:      labelsForCluster(cr),
-			Annotations: annotationsForCluster(cr),
```

Comment:
Q: We are dropping annotations on role binding.   I just want to make sure it is intentional.

Reply:
This is coming from the draft PR 
https://github.com/argoproj-labs/argocd-operator/pull/161

and I assume it is intentionally removed, may be they are not needed for rolebinding resources. I have added them for the clusterrolebindings.

Reply:
@sbose78 Is the removing the annotations above intentional?

Reply:
Honestly, I can't seem to remember why I had removed it :)

Reply:
Added it back @sbose78 @wtam2018